crates/macro-support/src/parser.rs

```diff
@@ -1559,14 +1560,14 @@ fn assert_no_lifetimes(sig: &syn::Signature) -> Result<(), Diagnostic> {
 }
 
 /// This method always fails if the BindgenAttrs contain variadic
-fn assert_not_variadic(attrs: &BindgenAttrs) -> Result<(), Diagnostic> {
-    if let Some(span) = attrs.variadic() {
-        let msg = "the `variadic` attribute can only be applied to imported \
-                   (`extern`) functions";
-        return Err(Diagnostic::span_error(*span, msg));
-    }
-    Ok(())
-}
+// fn assert_not_variadic(attrs: &BindgenAttrs) -> Result<(), Diagnostic> {
+//     if let Some(span) = attrs.variadic() {
+//         let msg = "the `variadic` attribute can only be applied to imported \
+//                    (`extern`) functions";
+//         return Err(Diagnostic::span_error(*span, msg));
+//     }
+//     Ok(())
+// }
```

Comment:
I think this can be removed now?

Reply:
:+1:

We shouldn't have dead code in there without a reason